src/System.Linq.Expressions/src/System/Dynamic/Utils/ExpressionUtils.cs

```diff
@@ -164,7 +164,7 @@ public static Expression ValidateOneArgument(MethodBase method, ExpressionType n
             {
                 pType = pType.GetElementType();
             }
-            TypeUtils.ValidateType(pType, methodParamName);
+            TypeUtils.ValidateType(pType, true, true, methodParamName);
```

Comment:
Speaking of which, can you use the optional parameter syntax for these calls so readers don't have to guess what the booleans mean?